packages/flutter/test/gestures/resampler_test.dart

```diff
@@ -153,6 +153,8 @@ void main() {
     expect(result[4].position.dy, 15.0);
     expect(result[4].delta.dx, 10.0);
     expect(result[4].delta.dy, -10.0);
+    // buttons field needs to be a valid value
+    expect(result[4].buttons != 0, true);
```

Comment:
```suggestion
    expect(result[4].buttons, kPrimaryButton);
```

Reply:
Maybe I can change it to
``` dart
expect(result[4].buttons > 0, true);
```
or
``` dart
expect(result[4].buttons == kPrimaryButton || result[4].buttons == kSecondaryButton || result[4].buttons == kTertiaryButton, true);
```
Because PointerUpEvent.buttons may not be kPrimaryButton.